browsers/opera_android.json

```
@@ -51,16 +52,19 @@
           "engine_version": "26"
         },
         "15": {
+          "release_date": "2013-07-02",
```

This one looks like another desktop release date. This page suggests July 8 2013

https://blogs.opera.com/news/2013/07/opera-15-for-android/

```suggestion
          "release_date": "2013-07-08",
          "release_notes": "https://blogs.opera.com/news/2013/07/opera-15-for-android/",
```